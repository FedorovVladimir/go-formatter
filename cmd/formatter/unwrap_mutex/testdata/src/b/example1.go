package main

import "sync"

func main() {
	mu := new(sync.Mutex) // want "unwrap_mutex"
	mu.Lock()
	defer mu.Unlock()
}
