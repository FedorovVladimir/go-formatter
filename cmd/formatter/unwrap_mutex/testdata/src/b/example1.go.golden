package main

import "sync"

func main() {
	var mu sync.Mutex // want "unwrap_mutex"
	mu.Lock()
	defer mu.Unlock()
}
